hosts

```diff
@@ -4,6 +4,10 @@ ny1.hashbang.sh
 sf1.hashbang.sh
 to1.hashbang.sh
 
+[shell_servers]
+ansible_user=root
```

Comment:
Arg.  Will do... in a separate PR?  :V

In general, though, there is no benefit not to do this: direct root isn't any more privileged than admin+sudo, and this avoids the dependency on LDAP/userdb.